### PR TITLE
Add minimal Mill Test Report

### PR DESCRIPTION
**Pull request by BenjaminMoe**
This pull request defines a new Minimal Mill Test Report credential. 

- All attributes are required
- All attributes are defined inline
- Schema is designed to be a 1:1 representation of an existing document

Checklist:

- [x] Minimal Mill Test Report Credential is defined
- [x] Mill Test Report Class is defined
- [x] Purchase Details Class is defined
- [x] Shipping Details Class is defined
- [x] Steel Chemical Analysis class is defined
- [x] Dimensions class is defined
- [x] Steel Mechanical class is defined
- [x] Steel Product Details class is defined
- [x] Examples are provided for each value
- [x] Placeholder terms are created for each term
- [ ] Issues are created to replace placeholder terms

### CODE REVIEW

**Comment by BenjaminMoe** [comment]
Migrated from: https://github.com/w3c-ccg/traceability-vocab/pull/692

**Comment by BenjaminMoe** [comment]
Not sure why I'm getting undefined terms. My plan is to create placeholder terms to make sure they are getting mapped through and then make issues to make sure they get updated. 

```json
{
   "@context":[
      "https://www.w3.org/2018/credentials/v1",
      "https://w3id.org/traceability/v1",
      {
         "@context":{
            "MinimalMillTestReportCredential":{
               "@context":{
                  
               },
               "@id":"https://w3id.org/traceability#MillTestReportCredential"
            },
            "SteelProductDetails":{
               "@context":{
                  "description":{
                     "@id":"https://w3id.org/traceability#description"
                  },
                  "grade":{
                     "@id":"https://w3id.org/traceability#grade"
                  },
                  "heatNo":{
                     "@id":"https://w3id.org/traceability#heatNo"
                  },
                  "otherGrades":{
                     "@id":"https://w3id.org/traceability#otherGrades"
                  },
                  "otherSpecifications":{
                     "@id":"https://w3id.org/traceability#otherSpecifications"
                  },
                  "quantity":{
                     "@id":"https://w3id.org/traceability#quantity"
                  },
                  "section":{
                     "@id":"https://w3id.org/traceability#section"
                  },
                  "standardGrades":{
                     "@id":"https://w3id.org/traceability#standardGrades"
                  },
                  "standardSpecifications":{
                     "@id":"https://w3id.org/traceability#standardSpecifications"
                  }
               },
               "@id":"https://w3id.org/traceability#SteelProductDetails"
            },
            "SteelChemicalAnalysis":{
               "@context":{
                  "C":{
                     "@id":"https://w3id.org/traceability#Carbon"
                  },
                  "Cb":{
                     "@id":"https://w3id.org/traceability#Cobalt"
                  },
                  "Cr":{
                     "@id":"https://w3id.org/traceability#Chromium"
                  },
                  "Cu":{
                     "@id":"https://w3id.org/traceability#Copper"
                  },
                  "Mn":{
                     "@id":"https://w3id.org/traceability#Manganese"
                  },
                  "Mo":{
                     "@id":"https://w3id.org/traceability#Molybdenum"
                  },
                  "Ni":{
                     "@id":"https://w3id.org/traceability#Nickel"
                  },
                  "P":{
                     "@id":"https://w3id.org/traceability#Phosphorus"
                  },
                  "S":{
                     "@id":"https://w3id.org/traceability#Sulfur"
                  },
                  "Si":{
                     "@id":"https://w3id.org/traceability#Silicon"
                  },
                  "Sn":{
                     "@id":"https://w3id.org/traceability#Tin"
                  },
                  "Ti":{
                     "@id":"https://w3id.org/traceability#Titanium"
                  },
                  "V":{
                     "@id":"https://w3id.org/traceability#Vanadium"
                  },
                  "cEq":{
                     "@id":"https://w3id.org/traceability#CarbonEquivalent"
                  }
               },
               "@id":"https://w3id.org/traceability#SteelChemicalAnalysis"
            },
            "Dimensions":{
               "@context":{
                  "height":{
                     "@id":"https://w3id.org/traceability#height"
                  },
                  "thickness":{
                     "@id":"https://w3id.org/traceability#thickness"
                  },
                  "width":{
                     "@id":"https://w3id.org/traceability#width"
                  }
               },
               "@id":"https://w3id.org/traceability#Dimensions"
            },
            "SteelMechanicalTests":{
               "@context":{
                  "charpyImpactEnergy":{
                     "@id":"https://w3id.org/traceability#charpyImpactEnergy"
                  },
                  "charpyImpactTem":{
                     "@id":"https://w3id.org/traceability#charpyImpactTem"
                  },
                  "dimensions":{
                     "@id":"https://w3id.org/traceability#dimensions"
                  },
                  "elangoration":{
                     "@id":"https://w3id.org/traceability#elangoration"
                  },
                  "netWeight":{
                     "@id":"https://w3id.org/traceability#netWeight"
                  },
                  "yieldStrength":{
                     "@id":"https://w3id.org/traceability#yieldStrength"
                  },
                  "yieldToTensileRatio":{
                     "@id":"https://w3id.org/traceability#yieldToTensileRatio"
                  }
               },
               "@id":"https://w3id.org/traceability#SteelMechanicalTests"
            },
            "PurchaseDetails":{
               "@context":{
                  "internalCertificateNumber":{
                     "@id":"https://w3id.org/traceability#internalCertificateNumber"
                  },
                  "invoiceNumber":{
                     "@id":"https://w3id.org/traceability#invoiceNumber"
                  },
                  "purchaseOrderNumber":{
                     "@id":"https://w3id.org/traceability#purchaseOrderNumber"
                  }
               },
               "@id":"https://w3id.org/traceability#PurchaseDetails"
            },
            "ShippingDetails":{
               "@context":{
                  "containerNumber":{
                     "@id":"https://w3id.org/traceability#containerNumber"
                  },
                  "customerAddress":{
                     "@id":"https://w3id.org/traceability#customerAddress"
                  },
                  "manufacturerAddress":{
                     "@id":"https://w3id.org/traceability#manufacturerAddress"
                  },
                  "masterBillOfLadingNumber":{
                     "@id":"https://w3id.org/traceability#masterBillOfLadingNumber"
                  }
               },
               "@id":"https://w3id.org/traceability#ShippingDetails"
            },
            "ContactPoint":{
               "@context":{
                  "email":{
                     "@id":"https://schema.org/email"
                  },
                  "jobTitle":{
                     "@id":"https://schema.org/jobTitle"
                  },
                  "name":{
                     "@id":"https://schema.org/name"
                  },
                  "phoneNumber":{
                     "@id":"https://schema.org/telephone"
                  },
                  "place":{
                     "@id":"https://w3id.org/traceability#place"
                  },
                  "worksFor":{
                     "@id":"https://schema.org/worksFor"
                  }
               },
               "@id":"https://schema.org/ContactPoint"
            },
            "Organization":{
               "@context":{
                  "address":{
                     "@id":"https://schema.org/PostalAddress"
                  },
                  "brand":{
                     "@id":"https://schema.org/Brand"
                  },
                  "contactPoint":{
                     "@id":"https://schema.org/ContactPoint"
                  },
                  "description":{
                     "@id":"https://schema.org/description"
                  },
                  "email":{
                     "@id":"https://schema.org/email"
                  },
                  "faxNumber":{
                     "@id":"https://schema.org/faxNumber"
                  },
                  "globalLocationNumber":{
                     "@id":"https://schema.org/globalLocationNumber"
                  },
                  "iataCarrierCode":{
                     "@id":"https://onerecord.iata.org/cargo/Company#airlineCode"
                  },
                  "legalName":{
                     "@id":"https://schema.org/legalName"
                  },
                  "leiCode":{
                     "@id":"https://schema.org/leiCode"
                  },
                  "location":{
                     "@id":"https://schema.org/location"
                  },
                  "name":{
                     "@id":"https://schema.org/name"
                  },
                  "phoneNumber":{
                     "@id":"https://schema.org/telephone"
                  },
                  "taxId":{
                     "@id":"https://schema.org/taxID"
                  },
                  "url":{
                     "@id":"https://schema.org/url"
                  }
               },
               "@id":"https://schema.org/Organization"
            },
            "MillTestReport":{
               "@context":{
                  "chemicalAnalysis":{
                     "@id":"https://w3id.org/traceability#chemicalAnalysis"
                  },
                  "countryOfMeltAndPour":{
                     "@id":"https://w3id.org/traceability#countryOfMeltAndPour"
                  },
                  "manufacturer":{
                     "@id":"https://w3id.org/traceability#manufacturer"
                  },
                  "mechanicalTests":{
                     "@id":"https://w3id.org/traceability#mechanicalTests"
                  },
                  "productDetails":{
                     "@id":"https://w3id.org/traceability#productDetails"
                  },
                  "purchaseDetails":{
                     "@id":"https://w3id.org/traceability#purchaseDetails"
                  },
                  "shippingDetails":{
                     "@id":"https://w3id.org/traceability#shippingDetails"
                  },
                  "steelLicenseNumber":{
                     "@id":"https://w3id.org/traceability#steelLicenseNumber"
                  }
               },
               "@id":"https://w3id.org/traceability#MillTestReport"
            }
         }
      }
   ],
   "type":[
      "VerifiableCredential",
      "MinimalMillTestReportCredential"
   ],
   "name":"Certified Mill Test Report",
   "description":"This document includes recommended mill Credential fields.",
   "issuer":"did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
   "issuanceDate":"2022-06-06T08:10:00+00:00",
   "credentialSubject":{
      "type":[
         "MillTestReport"
      ],
      "manufacturer":{
         "type":[
            "Organization"
         ],
         "name":"Melt Global, Inc.",
         "url":"https://melt-global.example.com",
         "phoneNumber":"+1 702-647-9292",
         "contactPoint":{
            "type":[
               "ContactPoint"
            ],
            "name":"Jon Doe",
            "jobTitle":"Cheif Inspector",
            "worksFor":"IRON APPROVERS INC."
         },
         "address":{
            "type":[
               "PostalAddress"
            ],
            "addressLocality":"Hunter Valley",
            "addressRegion":"Texas",
            "postalCode":"78599",
            "addressCountry":"United States"
         }
      },
      "countryOfMeltAndPour":"US",
      "steelLicenseNumber":"SF52W0720",
      "purchaseDetails":{
         "type":[
            "PurchaseDetails"
         ],
         "invoiceNumber":"I 20230211",
         "purchaseOrderNumber":"PO 13502441",
         "internalCertificateNumber":"131216-FO17-01/01"
      },
      "shippingDetails":{
         "type":[
            "ShippingDetails"
         ],
         "containerNumber":"SEA-01",
         "masterBillOfLadingNumber":"HASL6022D2CA001",
         "manufacturerAddress":{
            "type":[
               "PostalAddress"
            ],
            "addressLocality":"Hunter Valley",
            "addressRegion":"Texas",
            "postalCode":"78599",
            "addressCountry":"United States"
         },
         "customerAddress":{
            "type":[
               "PostalAddress"
            ],
            "addressLocality":"Sprague",
            "addressRegion":"Manitoba",
            "postalCode":"R0A1Z0",
            "addressCountry":"CA"
         }
      },
      "chemicalAnalysis":{
         "type":[
            "SteelChemicalAnalysis"
         ],
         "C":15.5,
         "Mn":0.5,
         "P":0.5,
         "Si":0.002,
         "Cu":0.14,
         "Cr":0.2,
         "Ni":3.4,
         "Mo":0.02,
         "Cb":1.2,
         "V":0.23,
         "Sn":0.25,
         "S":0.123,
         "Ti":1.25,
         "cEq":0.1
      },
      "mechanicalTests":{
         "type":[
            "SteelMechanicalTests"
         ],
         "dimensions":{
            "type":[
               "Dimensions"
            ],
            "width":"1.58m",
            "height":"0.3m",
            "thickness":"0.4m"
         },
         "netWeight":"0.25t",
         "yieldToTensileRatio":"712.3 MPA",
         "yieldStrength":"467.5 MPA",
         "elangoration":"250.51 MMT",
         "charpyImpactTem":"14 P1",
         "charpyImpactEnergy":"203 MMT"
      },
      "productDetails":{
         "type":[
            "SteelProductDetails"
         ],
         "heatNo":"225606",
         "section":"ASTM- A615-01a",
         "grade":"60",
         "quantity":"3",
         "description":"SS490 steel is a structural hot rolled sheet in the form of plates, sheets and strips for general structural applications. ",
         "otherSpecifications":"AST-51",
         "standardGrades":"SUS201",
         "standardSpecifications":"JIS G 3101",
         "otherGrades":"BF-4122"
      }
   },
   "proof":{
      "type":"Ed25519Signature2018",
      "created":"2023-01-23T16:36:30Z",
      "verificationMethod":"did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U#z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
      "proofPurpose":"assertionMethod",
      "jws":"eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..EfDg9XAG5ZPBbK2kJHNHMT2U1xE_gdS3jxoMOyLHu2aljdA6LuAz852pW-GSMPeF30QAulcOc1EjiyigxiweDg"
   }
}
```

**Comment by BenjaminMoe** [comment]
And here are the nquads

```
_:c14n0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/ContactPoint> .
_:c14n0 <https://schema.org/jobTitle> "Cheif Inspector" .
_:c14n0 <https://schema.org/name> "Jon Doe" .
_:c14n0 <https://schema.org/worksFor> "IRON APPROVERS INC." .
_:c14n1 <http://purl.org/dc/terms/created> "2023-01-23T16:36:30Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> _:c14n5 .
_:c14n1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/security#Ed25519Signature2018> _:c14n5 .
_:c14n1 <https://w3id.org/security#jws> "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..EfDg9XAG5ZPBbK2kJHNHMT2U1xE_gdS3jxoMOyLHu2aljdA6LuAz852pW-GSMPeF30QAulcOc1EjiyigxiweDg" _:c14n5 .
_:c14n1 <https://w3id.org/security#proofPurpose> <https://w3id.org/security#assertionMethod> _:c14n5 .
_:c14n1 <https://w3id.org/security#verificationMethod> <did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U#z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U> _:c14n5 .
_:c14n10 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#SteelMechanicalTests> .
_:c14n10 <https://w3id.org/traceability#charpyImpactEnergy> "203 MMT" .
_:c14n10 <https://w3id.org/traceability#charpyImpactTem> "14 P1" .
_:c14n10 <https://w3id.org/traceability#dimensions> _:c14n12 .
_:c14n10 <https://w3id.org/traceability#elangoration> "250.51 MMT" .
_:c14n10 <https://w3id.org/traceability#netWeight> "0.25t" .
_:c14n10 <https://w3id.org/traceability#yieldStrength> "467.5 MPA" .
_:c14n10 <https://w3id.org/traceability#yieldToTensileRatio> "712.3 MPA" .
_:c14n11 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/Organization> .
_:c14n11 <https://schema.org/ContactPoint> _:c14n0 .
_:c14n11 <https://schema.org/PostalAddress> _:c14n4 .
_:c14n11 <https://schema.org/name> "Melt Global, Inc." .
_:c14n11 <https://schema.org/telephone> "+1 702-647-9292" .
_:c14n11 <https://schema.org/url> "https://melt-global.example.com" .
_:c14n12 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#Dimensions> .
_:c14n12 <https://w3id.org/traceability#height> "0.3m" .
_:c14n12 <https://w3id.org/traceability#thickness> "0.4m" .
_:c14n12 <https://w3id.org/traceability#width> "1.58m" .
_:c14n13 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#MillTestReport> .
_:c14n13 <https://w3id.org/traceability#chemicalAnalysis> _:c14n9 .
_:c14n13 <https://w3id.org/traceability#countryOfMeltAndPour> "US" .
_:c14n13 <https://w3id.org/traceability#manufacturer> _:c14n11 .
_:c14n13 <https://w3id.org/traceability#mechanicalTests> _:c14n10 .
_:c14n13 <https://w3id.org/traceability#productDetails> _:c14n6 .
_:c14n13 <https://w3id.org/traceability#purchaseDetails> _:c14n3 .
_:c14n13 <https://w3id.org/traceability#shippingDetails> _:c14n7 .
_:c14n13 <https://w3id.org/traceability#steelLicenseNumber> "SF52W0720" .
_:c14n14 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/PostalAddress> .
_:c14n14 <https://schema.org/addressCountry> "CA" .
_:c14n14 <https://schema.org/addressLocality> "Sprague" .
_:c14n14 <https://schema.org/addressRegion> "Manitoba" .
_:c14n14 <https://schema.org/postalCode> "R0A1Z0" .
_:c14n2 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#MillTestReportCredential> .
_:c14n2 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/2018/credentials#VerifiableCredential> .
_:c14n2 <https://schema.org/description> "This document includes recommended mill Credential fields." .
_:c14n2 <https://schema.org/name> "Certified Mill Test Report" .
_:c14n2 <https://w3id.org/security#proof> _:c14n5 .
_:c14n2 <https://www.w3.org/2018/credentials#credentialSubject> _:c14n13 .
_:c14n2 <https://www.w3.org/2018/credentials#issuanceDate> "2022-06-06T08:10:00+00:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
_:c14n2 <https://www.w3.org/2018/credentials#issuer> <did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U> .
_:c14n3 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#PurchaseDetails> .
_:c14n3 <https://w3id.org/traceability#internalCertificateNumber> "131216-FO17-01/01" .
_:c14n3 <https://w3id.org/traceability#invoiceNumber> "I 20230211" .
_:c14n3 <https://w3id.org/traceability#purchaseOrderNumber> "PO 13502441" .
_:c14n4 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/PostalAddress> .
_:c14n4 <https://schema.org/addressCountry> "United States" .
_:c14n4 <https://schema.org/addressLocality> "Hunter Valley" .
_:c14n4 <https://schema.org/addressRegion> "Texas" .
_:c14n4 <https://schema.org/postalCode> "78599" .
_:c14n6 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#SteelProductDetails> .
_:c14n6 <https://w3id.org/traceability#description> "SS490 steel is a structural hot rolled sheet in the form of plates, sheets and strips for general structural applications. " .
_:c14n6 <https://w3id.org/traceability#grade> "60" .
_:c14n6 <https://w3id.org/traceability#heatNo> "225606" .
_:c14n6 <https://w3id.org/traceability#otherGrades> "BF-4122" .
_:c14n6 <https://w3id.org/traceability#otherSpecifications> "AST-51" .
_:c14n6 <https://w3id.org/traceability#quantity> "3" .
_:c14n6 <https://w3id.org/traceability#section> "ASTM- A615-01a" .
_:c14n6 <https://w3id.org/traceability#standardGrades> "SUS201" .
_:c14n6 <https://w3id.org/traceability#standardSpecifications> "JIS G 3101" .
_:c14n7 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#ShippingDetails> .
_:c14n7 <https://w3id.org/traceability#containerNumber> "SEA-01" .
_:c14n7 <https://w3id.org/traceability#customerAddress> _:c14n14 .
_:c14n7 <https://w3id.org/traceability#manufacturerAddress> _:c14n8 .
_:c14n7 <https://w3id.org/traceability#masterBillOfLadingNumber> "HASL6022D2CA001" .
_:c14n8 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://schema.org/PostalAddress> .
_:c14n8 <https://schema.org/addressCountry> "United States" .
_:c14n8 <https://schema.org/addressLocality> "Hunter Valley" .
_:c14n8 <https://schema.org/addressRegion> "Texas" .
_:c14n8 <https://schema.org/postalCode> "78599" .
_:c14n9 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://w3id.org/traceability#SteelChemicalAnalysis> .
_:c14n9 <https://w3id.org/traceability#Carbon> "1.55E1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#CarbonEquivalent> "1.0E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Chromium> "2.0E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Cobalt> "1.2E0"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Copper> "1.4E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Manganese> "5.0E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Molybdenum> "2.0E-2"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Nickel> "3.4E0"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Phosphorus> "5.0E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Silicon> "2.0E-3"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Sulfur> "1.23E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Tin> "2.5E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Titanium> "1.25E0"^^<http://www.w3.org/2001/XMLSchema#double> .
_:c14n9 <https://w3id.org/traceability#Vanadium> "2.3E-1"^^<http://www.w3.org/2001/XMLSchema#double> .
```

**Comment by BenjaminMoe** [comment]
Undefined term issue is fixed. I'll do a pass to look at the existing terms to see how many of the placeholders I can fix before making as `Ready for Review`.

File: docs/openapi/components/schemas/common/MillTestReport.yml [hunk 1]
```diff
@@ -16,44 +16,188 @@ properties:
...
+    },
+    "productDetails":{
+        "type":[
+          "SteelProductDetails"
+        ],
+        "heatNo":"225606",
+        "section":"ASTM- A615-01a",
+        "grade":"60",
+        "quantity":"3",
+        "description":"SS490 steel is a structural hot rolled sheet in the form of plates, sheets and strips for general structural applications. ",
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/MillTestReport.yml:197]:
```suggestion
        "description":"SS490 steel is a structural hot rolled sheet in the form of plates, sheets, and strips for general structural applications. ",
```

File: docs/openapi/components/schemas/common/ShippingDetails.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+$linkedData:
+  term: ShippingDetails
+  '@id': https://w3id.org/traceability#ShippingDetails
+title: Shipping Details
+description: Details for a shipment, such as the bill of lading number and
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/ShippingDetails.yml:5]:
```suggestion
description: Details for a shipment, such as the bill of lading number
```

File: docs/openapi/components/schemas/common/SteelProductDetails.yml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    type: string
+    example: 3
+    $linkedData:
+      term: quantity
+      '@id': https://w3id.org/traceability#quantity
+  description:
+    title: Descrption
+    description: Description of steel items
+    type: string
+    example: SS490 steel is a structural hot rolled sheet in the form of plates, sheets and strips for general structural applications.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/SteelProductDetails.yml:55]:
```suggestion
    example: SS490 steel is a structural hot rolled sheet in the form of plates, sheets, and strips for general structural applications. 
```

File: docs/openapi/components/schemas/common/SteelProductDetails.yml [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+example: |-
+  {
+    "type":[
+        "SteelProductDetails"
+    ],
+    "heatNo":"225606",
+    "section":"ASTM- A615-01a",
+    "grade":"60",
+    "quantity":"3",
+    "description":"SS490 steel is a structural hot rolled sheet in the form of plates, sheets and strips for general structural applications. ",
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/SteelProductDetails.yml:112]:
```suggestion
    "description":"SS490 steel is a structural hot rolled sheet in the form of plates, sheets, and strips for general structural applications. ",
```

File: docs/openapi/components/schemas/credentials/MinimalMillTestReportCredential.yml [hunk 5]
```diff
@@ -514,38 +567,47 @@ properties:
...
+            description: Quantity of steel items
             type: string
+            example: 3
           description:
-            title: Yield Strength
-            description: Yield Strength
+            title: Descrption
+            description: Description of steel items
             type: string
+            example: SS490 steel is a structural hot rolled sheet in the form of plates, sheets and strips for general structural applications.
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/MinimalMillTestReportCredential.yml:590]:
```suggestion
            example: SS490 steel is a structural hot rolled sheet in the form of plates, sheets, and strips for general structural applications. 
```

File: docs/openapi/components/schemas/credentials/MinimalMillTestReportCredential.yml [hunk 6]
```diff
@@ -752,18 +818,18 @@ example: |-
         "section": "ASTM- A615-01a",
         "grade": "60",
         "quantity": "3",
-        "description": "RF 600e plates",
-        "otherSpecifications": "NA",
-        "standardGrades": "NA",
-        "standardSpecifications": "NA",
-        "otherGrades": "NA"
+        "description": "SS490 steel is a structural hot rolled sheet in the form of plates, sheets and strips for general structural applications. ",
```
Comment (TallTed) [inline docs/openapi/components/schemas/credentials/MinimalMillTestReportCredential.yml:821]:
```suggestion
        "description": "SS490 steel is a structural hot rolled sheet in the form of plates, sheets, and strips for general structural applications. ",
```

**Comment by BenjaminMoe** [comment]
Thanks @TallTed. I got feedback that this was too much of a change and I'm moving this PR to https://github.com/w3c-ccg/traceability-vocab/pull/695.

**Pull request closed without merging**